site-service/src/main/resources/application-local.yml

```diff
@@ -6,6 +6,9 @@ server:
 fhir:
   uri: http://localhost:8099
   resultCount: 50
+role:
+  service:
+    uri: http://localhost:8082
```

Comment:
Is this config to make the authz on site-service work locally?  If so you may be able to remove this fix since I've just merged a PR which does the same thing, but in application.yml in the security package https://github.com/NDPH-ARTS/mts-services/commit/7d923a26d71a8c0f9ba07915d0552540184a28d1  Might be a different issue of course!

Reply:
yep, not needed